packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -34,8 +34,10 @@ export const createMuteState = (type, meeting, mediaDirection, sdkOwnsLocalTrack
    the last requested state by the client.
 
    More info about Locus muting API: https://sqbu-github.cisco.com/pages/WebExSquared/locus/guides/mute.html#
+
+   This class is exported only for unit tests. It should never be instantiated directly with new MuteState(), instead createMuteState() should be called
```

Comment:
you can add @internal tag

Reply:
I don't think it will help. "internal" tag hides the method from type declaration, so it can't be easily called from other packages, but it won't stop someone from calling new MuteState() from within plugin-meetings

Reply:
actually I'm wrong, "internal" can help even within plugin-meetings, but here we want the class just don't want to instantiate it with "new" so I guess we would have to set "internal" for the constructor? I tried it but then the class doesn't have the constructor at all in the d.ts file, is that ok? feels a bit weird...

Reply:
i think that's fine? that's the whole purpose to hide the instantiating i think

Reply:
anyway, not a big deal honestly